src/loaders/omexml.ts

```diff
@@ -47,7 +47,7 @@ export function fromString(str: string) {
         const sizes = (['X', 'Y', 'Z'] as const)
           .map(name => {
             const size = Pixels[`PhysicalSize${name}` as const];
-            const unit = Pixels[`PhysicalSize${name}Unit` as const];
+            const unit = Pixels[`PhysicalSize${name}Unit` as const] || 'µm'; // Not a required field
```

Comment:
This should be captured in the type itself below:

```typescript 
Attrs<PhysicalSizeUnit<'X' | 'Y' | 'Z'>, UnitsLength | undefined> &
```

and then here:

```typescript
const unit: UnitsLength = Pixels[`PhysicalSize${name}Unit` as const] ?? 'µm'; 
```

Reply:
Looks like my comment here did not go through but wouldn't we want the type definition to be 
```javascript
Attrs<PhysicalSizeUnit<'X' | 'Y' | 'Z'>, UnitsLength> &
```
since we are making sure it's not `undefined` in the `fromString` function?